vault/dendron.topic.special-notes.commands.create-daily-journal-note.md

```diff
@@ -0,0 +1,19 @@
+---
+id: 9stxbnf9hxmxt76se5dtx2b
+title: "Dendron: Create Daily Journal Note"
+desc: "Create a daily journal note."
+updated: 1661498852805
+created: 1661171483488
+---
+
+## Summary
+
+{{fm.desc}}
+
+## Keybindings
+- mac: `cmd+shift+i`
+- windows: `ctrl+shift+i`
+
+## Details
```

Comment:
i don't think details are necessary for this since its re-stating the title. on a second pass, we can show examples here